scripts/background.js

```diff
@@ -159,3 +160,18 @@ function icon_timer_updateBadge(tab_id, settings) {
         }
     });
 }
+
+// Handle messages sent by the extension to the runtime.
+chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
```

Comment:
This is not just extension, this is all messages going on in the browser, no? Why are we even adding this in this PR? this should have been a 1 line PR changing "cors" to "no-cors" (maybe 3 lines to make the call opaque).

Reply:
It's extension specific. The Firefox add-on debugger (at least for temporary) does not restrict what it displays to the debugged extension/add-on.

The message listener is added here because extension messaging is used by crown submission to report debugging level errors, and previously was not added - sent messages had no listener.